middleware/file/source/manager/resource.cpp

```diff
@@ -211,6 +211,20 @@ namespace casual::file::resource
 
             return result;
          }
+
+         namespace xa
+         {
+            namespace reverse
+            {
+               auto type( const auto& request)
+               {
+                  auto reply = common::message::reverse::type( request);
+                  reply.resource = request.resource;
+                  reply.trid = request.trid;
+                  return reply;
+               }
+            } // reverse
+         } // xa
```

Comment:
nitpick: `namespace xa::reverse` .
Otherwise, nice to have an abstraction for the 3 xa messages.